anta/tests/routing/isis.py

```diff
@@ -442,3 +442,89 @@ def _via_matches(self, via_input: TunnelPath, eos_via: dict[str, Any]) -> bool:
             and (via_input.interface is None or via_input.interface == eos_via.get("interface"))
             and (via_input.tunnel_id is None or via_input.tunnel_id.upper() == get_value(eos_via, "tunnelId.type", default="").upper())
         )
+
+
+class VerifyISISGracefulRestart(AntaTest):
+    """Verifies the graceful restart and helper mechanism.
+
+    This test performs the following checks:
+
+     1. Verifies that the ISIS is configured.
```

Comment:
Let's use `IS-IS` (with hyphen) to follow EOS doc and other tests. 
```suggestion
     1. Verifies that IS-IS is configured.
```